cfme/tests/infrastructure/test_vm_ownership.py

```diff
@@ -85,3 +91,35 @@ def test_set_vm_owner(self, appliance, vm, from_detail):
         rest_vm.reload()
         assert hasattr(rest_vm, "evm_owner_id")
         assert rest_vm.evm_owner.userid == "admin"
+
+
+@pytest.fixture(scope='module')
+def small_vm(provider, small_template_modscope):
+    vm = provider.appliance.collections.infra_vms.instantiate(random_vm_name(context='rename'),
+                                                              provider,
+                                                              small_template_modscope.name)
+    vm.create_on_provider(find_in_cfme=True, allow_skip="default")
+    vm.refresh_relationships()
+    yield vm
+    vm.cleanup_on_provider()
+
+
+@pytest.mark.ignore_stream('5.9')
+@pytest.mark.provider([VMwareProvider], override=True, scope="module", selector=ONE_PER_TYPE)
```

Comment:
Required:
Please, Provider scope should always `function` not `module`

Reply:
Has not been updated.

Reply:
I did not change because it was clashing with old implementations. Now I have modified and used ```small_template``` fixture which is of  **function** scope to get vm template and changed scope of ```small_vm`` fixture to **function** scope.
Now I can able to run this test with **function** scope.
Please let me know, if you any suggestions around it.
Thank you @mshriver @digitronik